frontend/scripts/react-components/dashboard-element/dashboard-widget/dashboard-widget.component.jsx

```diff
@@ -24,6 +24,15 @@ function DashboardWidget(props) {
 
   const [isModalOpen, openModal] = useState(false);
 
+  const openTableModal = useCallback(() => {
+    openModal(true);
+    openTableView(title);
```

Comment:
Shouldn't we specify that this is just for analytics?

Reply:
will do